src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingWithCustomSignatureMethod.cs

```diff
@@ -22,6 +22,10 @@ public class SigningAndVerifyingWithCustomSignatureMethod
 <test>some text node</test>
 </example>";
 
+        private static bool SupportsSha2Algorithms =>
+            !PlatformDetection.IsFullFramework ||
+            CryptoConfig.CreateFromName("http://www.w3.org/2001/04/xmldsig-more#rsa-sha384") as SignatureDescription != null;
```

Comment:
Don't we need to add a PlatformDetection.IsNet462OrNewer() as well? 

Something like: 
```cs
private static bool SupportsSha2Algorithms =>
                         !PlatformDetection.IsFullFramework ||
                         (PlatformDetection.IsNet462OrNewer() &&
                         CryptoConfig.CreateFromName("http://www.w3.org/2001/04/xmldsig-more#rsa-sha384") as SignatureDescription != null);
```

Because I see that the test is failing with:
```
System.Security.Cryptography.CryptographicException : SignatureDescription could not be created for the signature algorithm supplied.
```